src/store.js

```diff
@@ -289,7 +291,7 @@ function resetStoreVM (store, state, hot) {
         oldVm._data.$$state = null
       })
     }
-    Vue.nextTick(() => oldVm.$destroy())
+    Vue.nextTick(() => { oldVm.$destroy(); oldVm.computed = null })
```

Comment:
Why do you assign `null` to `oldVm.computed`?

Reply:
This was accidental and absolutely not required for this fix. Corrected.